src/Symfony/Bridge/Twig/Tests/Extension/FormExtensionTableLayoutTest.php

```diff
@@ -76,6 +76,34 @@ public function testStartTagHasActionAttributeWhenActionIsZero()
         $this->assertSame('<form name="form" method="get" action="0">', $html);
     }
 
+    public function testUrlWithDefaultProtocol()
```

Comment:
I don't think you should duplicate them in all components. Put them in AbstractLayoutTest instead, where the old test was.

Reply:
I guess @xabbuh had a reason?

Reply:
Well, the commit doing that says "fix deps=high". But I think that having them in AbstractLayoutTest will work fine for deps=high once the updated code is merged up to the highest version too.

Reply:
Reverting then.

Reply:
@MatTheCat I rebased and removed the commit entirely instead.